qemu/tests/usb_basic_check.py

```diff
@@ -126,14 +128,14 @@ def _check_dev():
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     timeout = float(params.get("login_timeout", 240))
-    error_context.context("Try to log into guest.", logging.info)
+    error_context.context("Try to log into guest.", test.log.info)
     session = vm.wait_for_login(timeout=timeout)
 
     error_context.context("Verify device(s) before rebooting.")
     _check_dev()
 
     if params.get("reboot_method"):
-        error_context.context("Reboot guest.", logging.info)
+        error_context.context("Reboot guest.", test.log.info)
```

Comment:
I want to comment L143, not this line.   Could you fix the issue together with this patch, add `test.log.info` in L143 ?

Reply:
After our discussion, will skip this suggestion as it seems to be the expected behavior, thanks.